tests/console/yast2_dns_server.pm

```diff
@@ -164,7 +171,7 @@ sub post_fail_hook {
     assert_script_run "tar cvf /tmp/dns_troubleshoot.tar @tar_input_files";
     upload_logs('/tmp/dns_troubleshoot.tar');
     $self->SUPER::post_fail_hook();
+
```

Comment:
added on purpose?

Reply:
Actually this one was added when I called tools/tidy:
```bash
--- ./tests/console/yast2_dns_server.pm 2018-07-23 16:46:34.249137898 +0200
+++ ./tests/console/yast2_dns_server.pm.tdy     2018-07-23 16:46:04.296938965 +0200
@@ -176,6 +176,7 @@
     assert_script_run "tar cvf /tmp/dns_troubleshoot.tar @tar_input_files";
     upload_logs('/tmp/dns_troubleshoot.tar');
     $self->SUPER::post_fail_hook();
+
 }
```
Looks like had some blank symbols there...